Main.py

```diff
@@ -103,93 +103,93 @@ def main(args, seed=None, baked_server_options: Optional[Dict[str, object]] = No
 
     # This assertion method should not be necessary to run if we are not outputting any multidata.
     if not args.skip_output:
-        AutoWorld.call_stage(world, "assert_generate")
+        AutoWorld.call_stage(multiworld, "assert_generate")
 
-    AutoWorld.call_all(world, "generate_early")
+    AutoWorld.call_all(multiworld, "generate_early")
 
     logger.info('')
 
-    for player in world.player_ids:
-        for item_name, count in world.worlds[player].options.start_inventory.value.items():
+    for player in multiworld.player_ids:
+        for item_name, count in multiworld.worlds[player].options.start_inventory.value.items():
             for _ in range(count):
-                world.push_precollected(world.create_item(item_name, player))
+                multiworld.push_precollected(multiworld.create_item(item_name, player))
 
-        for item_name, count in getattr(world.worlds[player].options,
+        for item_name, count in getattr(multiworld.worlds[player].options,
                                         "start_inventory_from_pool",
                                         StartInventoryPool({})).value.items():
             for _ in range(count):
-                world.push_precollected(world.create_item(item_name, player))
+                multiworld.push_precollected(multiworld.create_item(item_name, player))
             # remove from_pool items also from early items handling, as starting is plenty early.
-            early = world.early_items[player].get(item_name, 0)
+            early = multiworld.early_items[player].get(item_name, 0)
             if early:
-                world.early_items[player][item_name] = max(0, early-count)
+                multiworld.early_items[player][item_name] = max(0, early-count)
                 remaining_count = count-early
                 if remaining_count > 0:
-                    local_early = world.early_local_items[player].get(item_name, 0)
+                    local_early = multiworld.early_local_items[player].get(item_name, 0)
                     if local_early:
-                        world.early_items[player][item_name] = max(0, local_early - remaining_count)
+                        multiworld.early_items[player][item_name] = max(0, local_early - remaining_count)
                     del local_early
             del early
 
-    logger.info('Creating World.')
-    AutoWorld.call_all(world, "create_regions")
+    logger.info('Creating MultiWorld.')
```

Comment:
While this isn't necessarily wrong these messages are user facing so I wonder if they should be more specific or flavored. "Creating worlds" would be fine or maybe something like "populating worlds"? and on a completely unrelated note man we need way more logging

Reply:
I noticed that as well, but didn't have a good suggestion. "Populating worlds" sounds like it could be fill, so I don't think that's a good solution. "Creating regions" is what I would go for, but that's kind of out of the scope of this PR